addons/l10n_it_stock_ddt/report/l10n_it_ddt_report.xml

```diff
@@ -119,7 +119,14 @@
                                         <span t-field="move.product_uom" groups="uom.group_uom"/>
                                     </td>
                                     <td>
-                                        <t t-set="lst_price" t-value="move.product_id.lst_price * move.quantity_done"/>
+                                        <t 
+                                            t-set="lst_price"
+                                            t-value="o.sale_id.pricelist_id.get_product_price(
+                                                move.product_id, 
+                                                move.quantity_done,
```

Comment:
`get_product_price` will only give the unit price of one product (but needs the quantity to know the right pricelist rule to apply).

Reply:
Thanks for taking the time to looking into it.

> Honestly, recomputing the price on the rendering of the report seems wrong to me.

You're right, there was a much simpler solution.

> you're using sale logic in a report of a module not depending on sale in any way...

`l10n_it_stock_ddt` depends on `delivery`, which depends on `sale_stock`.

Reply:
Yeah I think it would be safer. 
Thanks again 🙏🏾 .